grype/db/v6/affected_package_store.go

```diff
@@ -529,6 +516,60 @@ func (s *affectedPackageStore) applyAlias(d *OSSpecifier) error {
 	return nil
 }
 
+func (s *affectedPackageStore) searchForDistroVersionVariants(query *gorm.DB, d OSSpecifier) ([]OperatingSystem, error) {
```

Comment:
note: this function only migrated to make it more readable (in order of calls from parent functions).